src/simularium/VisGeometry.ts

```diff
@@ -1489,6 +1231,93 @@ class VisGeometry {
         return agent;
     }
 
+    private renderPdb(typeId: number, visAgent: VisAgent, pdbEntry: PDBModel) {
```

Comment:
these are unchanged, I just wanted to be able to see what was happening in updateScene more clearly